contributors/design-proposals/seccomp.md

```diff
@@ -214,6 +248,192 @@ profiles using the key
 `seccomp.security.alpha.kubernetes.io/allowedProfileNames`.  The value of this
 key should be a comma delimited list.
 
+### Spec
+
+We will start from the OCI specification. This API resource will be added to
+`settings.k8s.io` as an `alpha` resource.
+
+```
+// Seccomp represents syscall restrictions
```

Comment:
It's not clear whether this is global or namespaced - seems like namespaced?

Reply:
Is it possible for resources to be both namespaced & unnamespaced?

Reply:
> Is it possible for resources to be both namespaced & unnamespaced?

no